lib/galaxy/tool_util/xsd/galaxy.xsd

```diff
@@ -2393,9 +2393,25 @@ In your command template, you'll need to include the section name to access the
 variable:
 
 ```
---color $adv.plot_color
+$section_name.parameter_name
 ```
 
+In output filters sections are represented as dictionary with the same name as the section:
+
+```
+<filter>section_name['parameter_name']</filter>
+```
+
+In order to reference parameters in sections from tags in the `<outputs>` section, e.g. in the `format_source` attribute of `<data>` tags, the syntax is:
+
+```
+section_name|parameter_name
+```
+
+Until profile 21.01 `parameter_name` was sufficient (https://github.com/galaxyproject/galaxy/pull/9493/files).
+
+Note that references to other parameters in the `<inputs>` section are only possible if the reference is in the same section or its parents, therefor only `parameter_name` is used. 
```

Comment:
Can you provide an example here?

In theory we could have a parameter `foo` in the top-level inputs __and__ a parameter `foo` in a section. Do we?

Reply:
> Can you provide an example here?

Yes. 

> In theory we could have a parameter foo in the top-level inputs and a parameter foo in a section. Do we?

Yes, but this is discouraged ... nevertheless it might occur in automatic tool building (e.g. in OpenMS I had the case, but luckily no cross reference was needed). Wondering if we should file an issue for this?